objectron_dataset_detection.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python3
+
+__all__ = ['Objectron']
+
+import os
+import sys
+import io
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List, Tuple, Dict
+from PIL import Image
+import numpy as np
+import pickle
+
+# torch+torchvision
+import torch as th
+import torch.nn as nn
+import torchvision.io as thio
+from torchvision import transforms
+
+import torch_xla.utils.tf_record_reader as tfrr
+from google.cloud import storage
+
+
+NUM_KEYPOINTS = 9
+
+
+def _glob_objectron(bucket_name: str, prefix: str):
```

Comment:
duplicated code from `objectron_dataset.py` - maybe we should (*in the future*) refactor this out into something like `data/objectron_utils.py`. Out of scope for this PR though, so it's fine to leave as-is.

Reply:
That sounds like a better structure.